src/nimvscode.nim

```diff
@@ -313,7 +357,9 @@ proc activate*(ctx: VscodeExtensionContext): void =
   processConfig(config)
   discard vscode.workspace.onDidChangeConfiguration(configUpdate)
 
-  if config.getBool("enableNimsuggest"):
+  if config.getBool("enableLsp"):
+    startLanguageServer(true)
+  elif config.getBool("enableNimsuggest"):
```

Comment:
We shouldn't have two separate flags, just one property which is an enum:
- nimsuggest
- nimls
- none

Reply:
won't that be a breaking change?

Reply:
I'm trying to recall where I saw it, but there is a way to handle migration of config in vscode such that it's not. 🤔

Reply:
I can make it like that:

have a property `backend` which can be the 3 values listed above. The default is nimsuggest and the when `backend == nimsuggest && enableNimsuggest == false` assume that backend is `none`

Reply:
The value property works, let's not call it backend as that is already a property. Otherwise sounds good.